maven-embedder/src/main/java/org/apache/maven/cli/jansi/Style.java

```diff
@@ -28,6 +28,7 @@
  * @since 4.0.0
  */
 enum Style {
+    TRACE("bold,magenta"),
```

Comment:
This should be last to avoid changing existing ordinals in an incompatible way.

Reply:
It's not an existing, one.  It only has been introduced in 4.0, so it should be safe here.